index.js

```diff
@@ -22,7 +22,7 @@ const options = {
 		process.env.BOWER_REGISTRY_URL || 'http://origami-bower-registry.ft.com',
 	tempdir: `/tmp/buildservice-${process.pid}/`,
 	testHealthcheckFailure: process.env.TEST_HEALTHCHECK_FAILURE || false,
-	npmRegistryURL: process.env.NPM_REGISTRY_URL || 'https://registry.npmjs.com'
+	npmRegistryURL: process.env.NPM_REGISTRY_URL || 'https://origami-npm-registry-prototype.herokuapp.com/'
```

Comment:
we can put this back now?
```suggestion
	npmRegistryURL: process.env.NPM_REGISTRY_URL || 'https://registry.npmjs.com'
```